plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeHistoryTable.java

```diff
@@ -92,6 +93,7 @@ protected List<Page> buildPages(ConnectorSession session, PageListBuilder pagesB
             pagesBuilder.beginRow();
 
             pagesBuilder.appendBigint(commitInfoEntry.version());
+            commitInfoEntry.inCommitTimestamp().ifPresentOrElse(inCommitTimestamp -> pagesBuilder.appendTimestampTzMillis(inCommitTimestamp, timeZoneKey), pagesBuilder::appendNull);
             pagesBuilder.appendTimestampTzMillis(commitInfoEntry.timestamp(), timeZoneKey);
```

Comment:
I suspect this should be UTC too

Reply:
It's not in this pr scope. I would like to discuss it in another issue/pr separately